src/EnrollWindowsHelloController.js

```diff
@@ -110,25 +126,75 @@ function (Okta, FormController, FormType, webauthn, Footer, WindowsHelloErrorMes
 
       formChildren: function () {
         var result = [];
+
         if (!webauthn.isAvailable()) {
           result.push(
             FormType.View(
-              { View: WindowsHelloErrorMessageView },
-              { selector: '.o-form-error-container' }
+              { View: new WindowsHelloErrorMessageView(
+                { message: Okta.loc('enroll.windowsHello.error.notWindows', 'login') }) },
+              { selector: '.o-form-error-container'}
             )
           );
         }
+        else {
+          result.push(FormType.View({
```

Comment:
@rchild-okta @mauriciocastillosilva-okta @alexeystadnikov-okta 

I spoke with the UX team and the error notification belongs below the widget header and above any descriptive text.

After talking to @rchild-okta I suggest implementing the notification consistent with the current pattern and I will open a Jira for the fix.